helm-chart/splunk-kubernetes-metrics/templates/configMap.yaml

```diff
@@ -76,7 +95,7 @@ data:
       {{- if or .Values.splunk.hec.caFile .Values.global.splunk.hec.caFile }}
       ca_file /fluentd/etc/splunk/hec_ca_file
       {{- end }}
-      interval {{ or .Values.metricsInterval .Values.global.metricsInterval }}
+      interval 15s
```

Comment:
Restore previous functionality, you removed a configurable.

Reply:
done